.dependabot/config.yml

```diff
@@ -80,9 +80,6 @@ update_configs:
     update_schedule: "daily"
 
 # Examples
-  - package_manager: "java:maven"
+  - package_manager: "java:gradle"
```

Comment:
why did you remove `automerged_updates`? :)

Reply:
whoooooops